bazel/external/boringssl_fips.patch

```diff
@@ -0,0 +1,15 @@
+# Fix FIPS build (from BoringSSL commit 4ca15d5dcbe6e8051a4654df7c971ea8307abfe0).
+--- boringssl/util/fipstools/acvp/modulewrapper/modulewrapper.cc
++++ boringssl/util/fipstools/acvp/modulewrapper/modulewrapper.cc
+@@ -12,9 +12,11 @@
+  * OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
+  * CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE. */
+ 
++#include <string>
+ #include <vector>
+ 
+ #include <assert.h>
++#include <errno.h>
+ #include <string.h>
+ #include <sys/uio.h>
+ #include <unistd.h>
```

Comment:
@agl does this patch break FIPS validation or can we include it?

(We build against libc++, and FIPS build fails without those headers)

Reply:
Update LGTM; can you decide on this question and we can merge?

Reply:
modulewrapper.cc is not part of the FIPS module so can be patched without concern. (Indeed, you don't even need to build it: it's a utility for certification testing.)

(I'll make those changes to BoringSSL to avoid the need for this in the future.)

Reply:
(Actually, those changes have already been made.)

Reply:
@PiotrSikora can you either update the patch with comment (says it doesn't break FIPS validation), or not even build this?

Reply:
Sorry, I completely missed the response from @agl. Added comment to the patch. Thanks!